clarity_ext/service/step_logger_service.py

```diff
@@ -0,0 +1,60 @@
+import logging
+import time
+from clarity_ext.service.file_service import SharedFileNotFound
+
+
+class StepLoggerService:
+    """
+    The logger service provides logging for files attached in the step.
+
+    Extension writers are exposed to this class via the `ExtensionContext`, which supplies direct
+    access to both logging artifacts, without them having to do anything but ensuring that
```

Comment:
the word 'both' seem redundant in 'access to both logging', or am I missing something?